app/controllers/api/teams.php

```diff
@@ -471,6 +471,83 @@
         ;
     });
 
+App::patch('/v1/teams/:teamId/memberships/:membershipId')
+    ->desc('Update Membership Roles')
+    ->groups(['api', 'teams'])
+    ->label('event', 'teams.memberships.update')
+    ->label('scope', 'public')
+    ->label('sdk.auth', [APP_AUTH_TYPE_SESSION, APP_AUTH_TYPE_KEY])
```

Comment:
I believe `APP_AUTH_TYPE_JWT` is also required?

Reply:
Good catch!

Reply:
👍